src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/NamespaceCache.java

```diff
@@ -8,10 +8,14 @@
 import io.stargate.sgv2.jsonapi.api.request.DataApiRequestInfo;
 import io.stargate.sgv2.jsonapi.exception.ErrorCode;
 import io.stargate.sgv2.jsonapi.exception.JsonApiException;
+import io.stargate.sgv2.jsonapi.service.FeatureFlags;
 import io.stargate.sgv2.jsonapi.service.schema.model.JsonapiTableMatcher;
 import java.time.Duration;
 
 /** Caches the vector enabled status for the namespace */
+// TODO: what is the vector status of a namespace ? vectors are per collection
+// TODO: clarify the name of this class, it is a cache of the collections/ tables not a cache of
```

Comment:
+1 for renaming (although not 100% what the new name should be): it's caching Collections/Tables within a single Namespace.

Reply:
Possible future change, skipping for now.